account_invoice_inter_company/models/account_move.py

```diff
@@ -197,6 +197,24 @@ def _prepare_invoice_data(self, dest_company):
         )
         return vals
 
+    def button_draft(self):
+        for move in self:
+            company = move._find_company_from_invoice_partner()
```

Comment:
Why the need of searching for the company? Isn't `move.auto_generated` enough to know that a company is linked?

Reply:
Well, indeed that `auto_generated` field is not very useful, as we can deduce that from the `auto_invoice_id` already.